Menu/GameShell/10_Settings/Wifi/wifi_list.py

```diff
@@ -60,7 +60,7 @@ def Draw(self):
         self.Reset()
         
 class WifiInfoPage(Page):
-    _FootMsg =  ["Nav","","Disconnect","Back",""]
+    _FootMsg =  ["Nav","","","Back",""]
```

Comment:
I removed the text "Disconnect" from the array index since it's written down below on the "OnLoadCb" function, when it detects you are connected to that wireless network.

After thinking about it more carefully, I think it could be better to leave it as it was because it helps to know the function of that button at a first sight.